Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging.py

```diff
@@ -0,0 +1,285 @@
+                           headers=['TimeStamp', 'Log Name', 'Insert ID', 'Principal Email',
+                                    'Type', 'Project ID', 'Cluster Name', 'Service Name'],
+                           removeNull=True)
+
+
+def log_entries_list_command(client: GoogleCloudLoggingClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Gets lists log entries. Use this method to retrieve log entries that originated from a
+       project/folder/organization/billing account.
+
+    Args:
+        client (Client): A Client class to wrap the google cloud api library.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults containing the lists log entries.
+    """
+    resource_project_name = argToList(args.get('project_name', []))
+    resource_organization_name = argToList(args.get('organization_name', []))
+    resource_billing_account_name = argToList(args.get('billing_account_name', []))
+    resource_folders_names = argToList(args.get('folders_names', []))
+
+    if not (resource_project_name or resource_organization_name or resource_billing_account_name or resource_folders_names):
+        raise DemistoException('At least one resource from project_name, '
+                               'organization_name, billing_account_name, or folder_name must be provided.')
+    resources = []
+    for project_name in resource_project_name:
+        resources.append(f'projects/{project_name}')
+    for organization_name in resource_organization_name:
+        resources.append(f'organizations/{organization_name}')
+    for billing_account_name in resource_billing_account_name:
+        resources.append(f'billingAccounts/{billing_account_name}')
+    for folders_names in resource_folders_names:
+        resources.append(f'folders/{folders_names}')
+
+    limit = arg_to_number(args.get('limit'))
+    page_size = arg_to_number(args.get('page_size')) or 50
+    request_body = {'resourceNames': resources,
+                    'filter': args.get('filter'),
+                    'orderBy': args.get('order_by')}
+    if page_size and args.get('next_token') and not arg_to_number(args.get('limit')):
+        request_body['pageSize'] = page_size
+        request_body['pageToken'] = args.get('next_token')
+    else:
+        request_body['pageSize'] = limit or 50
+    response = {}
+    try:
+        if limit and limit > 1000:
+            # If the pageSize value is negative or exceeds 1000, the request is rejected.
+            request_body['pageSize'] = 1000
+            response = get_all_results(client, limit, request_body)
+        else:
+            demisto.debug(f'Request body: {request_body}')
+            response = client.get_entries_request(request_body)
+    except ValueError as e:
+        raise ValueError(e) from e
+    return CommandResults(
+        outputs_key_field='insertId',
+        outputs={'GoogleCloudLogging(true)': {'nextPageToken': response.get('nextPageToken')},
```

Comment:
```suggestion
        outputs={'GoogleCloudLogging(true)': {'nextPageToken': response.get('nextPageToken')},
```
(true)?

Reply:
It does not appear in the context. It is only used to rewrite the nextPageToken in the context when rerun the command